Assets/Scripts/GpsSensor.cs

```diff
@@ -62,11 +64,14 @@ private void ReportPosition()
             transform.position.x + _noise * Utilities.GaussianRandom()},
             new double[] {initLat, initLon});
 
+        double altitude = transform.position.y + _noise * Utilities.GaussianRandom();
```

Comment:
make sure to add `initAlt`

Reply:
I just realized that initAlt is not nescescary for position Y so could we remove it or is it used for some post processing stuff?